generate_status_quo.py

```diff
@@ -399,7 +428,8 @@ def make_spec(self):
             elif 'contrib/inventory' in spec_path:
                 spec_path = os.path.join('contrib', 'inventory', os.path.basename(spec_path))
             else:
-                import epdb; epdb.st()
+                #import epdb; epdb.st()
```

Comment:
FYI you could use
```suggestion
                breakpoint()
```
and then you'd run `PYTHONBREAKPOINT=0 python -m generate_status_quo` to disable it or `PYTHONBREAKPOINT=epdb.st python -m generate_status_quo` to automatically use your favorite debugger tool.